<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObCil</class>
 <widget class="QWidget" name="ObCil">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>585</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Цилиндрическая обечайка</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,0,0,0,0,300">
   <item row="5" column="2" colspan="3">
    <widget class="QComboBox" name="steel_cbob"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_29">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;К-т прочности сварного шва, φ&lt;span style=&quot; vertical-align:sub;&quot;&gt;p&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="c1_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLineEdit" name="c3_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>°С</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="label_34">
     <property name="text">
      <string>Общая прибавка, с</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_45">
     <property name="text">
      <string>Расчетная длина обечайки, l</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>МПа</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pbShowSxemaob">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Расчетная схема цилиндрической обечайки</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLineEdit" name="c_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Принятая толщина, s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="sigma_leob">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="fi_le">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>ГОСТ 34233.2-2017</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="label_33">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Технологическая прибавка, c&lt;span style=&quot; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="pbGetEob">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="pbfiob">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="dia_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="5">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="l_name">
     <property name="text">
      <string>Название элемента:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>l_name</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Прибавка на коррозию, c&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>МПа</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>Внутренний диаметр обечайки, D</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="label_32">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Минусовой допуск, c&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Допускаемое напряжение, [σ]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="E_leob">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pbGetSigmaob">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>[σ]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Расчетная температура, t&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QPushButton" name="pbGetlob">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLineEdit" name="name_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QLineEdit" name="s_leob"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="l_steel">
     <property name="text">
      <string>Марка материала</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="c2_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="3">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Нормативный документ:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="l_leob">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_44">
     <property name="text">
      <string>Модуль продольной упругости, E</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="temp_leob">
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QGroupBox" name="obpress_gb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>               Расчетное давление,  p</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="vn_rbob">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>внутренние</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="nar_rbob">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>наружное</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="press_leob">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>МПа</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="5" rowspan="15">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="minimumSize">
         <size>
          <width>269</width>
          <height>181</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>181</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>269</width>
          <height>181</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>pic/ObCil.gif</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="obnag_gb">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Нагрузки</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="nagobcalc_rb">
           <property name="text">
            <string>Определять при расчете</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="nagobhand_rb">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Задавать вручную</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="obf_gb">
        <property name="title">
         <string>Расчетное осевое усилие, F</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QRadioButton" name="rbf_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rbf_1">
           <property name="text">
            <string>1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="rbf_4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="rbf_3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="rbf_5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="rbf_6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="rbf_7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="4">
          <widget class="QLabel" name="label_rbf">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>pic/PC/PC1.gif</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="18" column="0" rowspan="2" colspan="6">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pbPredob">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>102</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Предварительный 
 расчет</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbobtoobyk">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>127</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Передать в
 укрепление отверстия</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCalcob">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Расчет</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCancelob">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="s_calc_lob">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="press_d_lob">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>p</string>
      </property>
     </widget>
     <widget class="QLabel" name="s_calc_lob_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="press_d_lob_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>p</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="17" column="0" colspan="6">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Дефекты по ГОСТ 34233.11-2017</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Изоляция и футеровка &gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Малоцикловая прочность &gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
